data/i18n/en/en.toml

```diff
@@ -249,7 +249,6 @@ other = """This section links to third party projects that provide functionality
 [thirdparty_message_edit_disclaimer]
 other="""Third party content advice"""
 
-
```

Comment:
Hi @shurup . Thanks for contributing.
I think the extra, empty line does not affect the file. Also, we recommend that you only include changes in a PR from one language.

cc @kubernetes/sig-docs-ko-reviews @kubernetes/sig-docs-zh-reviews

Reply:
Thanks for the feedback! I completely agree it doesn't really affect the file. My point was: when you start copying this file to other localizations and then comparing your localized version with the original one (to make sure you've covered every line in your translation) it's handier not to have any empty lines as they might be confusing when it comes to the line-by-line comparison (or based on the number of lines).

Reply:
+1 to what @kbhawkey said. I also think it'd be best if we delink the other languages from the PR as of now and decide if we find any value in actually moving forward with this one. If the changes submitted in the PR are accepted, you may open PRs for the other localizations. What do you think of that, @shurup?

Reply:
/cc @kubernetes/sig-docs-ko-reviews @kubernetes/sig-docs-zh-reviews

Reply:
@divya-mohan0209 sure, it's easy to return to my initial commit (affecting `en.toml` only). I did not want to make "a big thing" of it (i.e., multiple PRs just to remove one empty line…) at all. I mean, it's the tiniest step to bring a bit (literally, bit :-)) more order to the file that doesn't affect the actual content in translations (hence it wasn't evident to me that I needed to bring the attention of localization teams for that). When I was working on syncing the Russian translation of this file (#35019), it seemed natural to me to make this tiny fix: since it helped me with this syncing, I assumed it would be useful for others as well. If we don't feel we need that, let's just close this PR not to spend the precious time of the community on this small thing — I would be totally fine with that. And I am really sorry for this extra fuss.